engine/src/main/java/org/destinationsol/assets/Assets.java

```diff
@@ -40,6 +43,8 @@
 public abstract class Assets {
     private static AssetHelper assetHelper;
     private static Set<ResourceUrn> textureList;
+    // This map is initialized after the player selects their ship, it maps the human readable names of ships to their internal URIs
+    public static Map<String, String> playerSpawnConfigIdMap = new HashMap<String, String>();
```

Comment:
I like the idea of having this map, but I don't like the idea of having it here. `Assets` is supposed to be a helper class to assist in loading assets. So, what I would suggest is that you move this map to a different location - maybe in NewShipScreen for the time being?